toolkit/themes/shared/design-system/config.js

```diff
@@ -24,6 +49,7 @@ const MEDIA_QUERY_PROPERTY_MAP = {
  */
 function hcmFormatter(args) {
   return (
+    customFileHeader() +
```

Comment:
nice! this seems way better than having to reference the `fileHeader` in three different places when we have multiple formats 😄